### PR TITLE
[flutter_tts] Adds flutter_tts_tizen package

### PR DESCRIPTION
**Pull request by rookiejava**
This PR adds tizen implementation of [flutter_tts](https://github.com/dlutton/flutter_tts).

The features supported by Tizen are as follows. Other features are not supported due to platform limitation.
 - [x] speak
 - [x] stop
 - [x] pause
 - [x] get languages
 - [x] set language
 - [x] set speech rate

Tested on the following profiles and emulators/devices.
- Mobile profile
  - [x] emulator (4.0 or higher)
  - [x] TM1 (4.0 or higher)
- TV profile
  - [ ] emulator (4.0 or higher) -> It seems to workm but there is no sound.
  - [x] Samsung Smart TV (4.0 or higher)  
- Wearable profile
  - [x] emulator (4.0 or higher)
  - [x] Galaxy Watch Series (4.0 or higher)

   
flutter_tts_tv

### CODE REVIEW

**Comment by rookiejava** [comment]
Can someone take a look this [error](https://github.com/flutter-tizen/plugins/pull/66/checks?check_run_id=2339294336)? 

It occurs while verifying formatting as below:
```
Unchanged webview_flutter/example/test_driver/integration_test.dart
Unchanged webview_flutter/lib/webview_flutter_tizen.dart
Unchanged wifi_info_flutter/example/integration_test/wifi_info_test.dart
Unchanged wifi_info_flutter/example/lib/main.dart
Unchanged wifi_info_flutter/example/test_driver/integration_test.dart
Unchanged wifi_info_flutter/example/test_driver/integration_test.dart
Error: Process completed with exit code 1.
```

**Review by swift-kim — COMMENTED** [review]
Just some nitpicks. Thank you very much for your awesome work!

File: packages/flutter_tts/LICENSE [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Copyright (c) 2020 Samsung Electronics Co., Ltd. All rights reserved.
+Copyright (c) 2013 The Flutter Authors. All rights reserved.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
```
[the remark this answers is not in the document]
Comment (rookiejava) [reply packages/flutter_tts/LICENSE]: done!

File: packages/flutter_tts/example/integration_test/flutter_tts_test.dart [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```
[the remark this answers is not in the document]
Comment (rookiejava) [reply packages/flutter_tts/example/integration_test/flutter_tts_test.dart]: done!

File: packages/flutter_tts/example/.metadata [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# This file tracks properties of this Flutter project.
+# Used by Flutter tool to assess capabilities and perform upgrades etc.
+#
+# This file should be version controlled and should not be manually edited.
+
+version:
+  revision: c5a4b4029c0798f37c4a39b479d7cb75daa7b05c
+  channel: unknown
+
+project_type: app
```
[the remark this answers is not in the document]
Comment (rookiejava) [reply packages/flutter_tts/example/.metadata]: if so, why don't you adding .metadata to .gitignore. Anyway I've removed.

[the remark this answers is not in the document]
Comment (swift-kim) [reply packages/flutter_tts/example/.metadata]: The rule applies to this repo only and it's not the case for other app projects. Sorry to confuse you!

File: packages/flutter_tts/tizen/src/flutter_tts_tizen_plugin.cc [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+          LOG_ERROR("[TTS] tts_add_text failed: %s", ErrorToString(ret).c_str());
+          result->Error(std::to_string(ret), "Failed to speak(tts_add_text).");
+          return;
+        } 
+      } else {
+        result->Error("Invalid Arguments", "Invalid Arguments");
+        return;
+      }
+      ret = tts_play(m_tts);
+		  if (ret != TTS_ERROR_NONE) {
```
Comment (swift-kim) [inline packages/flutter_tts/tizen/src/flutter_tts_tizen_plugin.cc:185]: Tabs!

```suggestion
      if (ret != TTS_ERROR_NONE) {
```

**Review by swift-kim — APPROVED** [review]
Otherwise LGTM.

**Review by bbrto21 — APPROVED** [review]
LGTM

**Pull request merged**